### ISSUE TITLE
Na androide nejde přidat účastníka akce, přihlášku, účtování… prostě tam úplně chybí ta spodní nabídka

### ISSUE THREAD

**Issue by Grejdy** [bosancz/interni-sekce#41]
Je možný, že je tam prostě jenom schovaná pod okrajem obrazovky, ale to nic nemění na tom, že se k ní člověk nedostane.

**Zařízení:**
- Zařízení: mobil (androide - jak je to na iphonu netuším)
- Prohlížeč: Firefox (a možná i ostatní, to jsem nezkoušel)
- Zobrazení: normální i „stránky pro počítač“

**Comment by DavidTvrdy** [comment]
Já mám spodní nabídku u akce v pohodě

**Comment by Grejdy** [comment]
Jo, co jsem tak zkoušel, tak webové rozhraní v jakékoli podobě a to, co je založené na chromiu, je v tomhle ohledu bezproblémové. Zatím jediný případ, kde jsem to objevil, je když to má člověk stažené jako aplikaci přes firefox.
(Takže to není tak velký problém, jak jsem myslel, ale pokud to má nějaké jednoduché řešení - třeba doplnit chybějící řádek v nějakém css, nebo tak něco - tak by nemuselo být od věci to přihodit do nějakého dalšího updatu. Ale jestli to bude složitější, tak se tím asi nemá moc cenu zabývat.)
